src/test/resources/features/contactUs/contactUs.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Contact Us
+
+  Scenario: User is able to send a message correctly
+    Given User is on contact us page
+    When he fills fields correctly
+    Then Message is send
+
+    Scenario: User doesn't send the message
+      Given User is on contact us page
+      When he doesn't fill every field
+      Then Message is not send
```

Comment:
@ Given User is on contact us page - user juz tu powinien byc z małej litery. tak samo Message po then